backend/core/utils/time.py

```diff
@@ -9,3 +9,9 @@ def time_ago(**kwargs):
     """Return a time in the past as specified as keyword arguments."""
 
     return timezone.now() - timedelta(**kwargs)
+
+
+def time_ahead(**kwargs):
```

Comment:
The following suggestion requires importing datetime in addition to timedelta.

```suggestion
def time_ahead(**kwargs) -> datetime:
```